src/sentry/runner/initializer.py

```diff
@@ -188,14 +188,15 @@ def configure_structlog():
 
     structlog.configure(**kwargs)
 
-    lvl = os.environ.get('SENTRY_LOG_LEVEL', None)
+    lvl = os.environ.get('SENTRY_LOG_LEVEL', 'INFO')
```

Comment:
And we should be abel to guarantee that this env variable was set becasue the runner should be setting it.